### ISSUE TITLE
Error importing platform config_flow from integration heatzy to set up heatzy configuration entry: No module named 'heatzypy' (v.2024.1.0 &  ws-6.1.0)

### ISSUE THREAD

**Issue by luke7101** [cyr-ius/hass-heatzy#53]
Heatzy custom integration failed to setup

Home Assistant v.2024.1.0
hass-heatzy ws-6.1.0

```
2024-01-04 11:27:47.659 ERROR (MainThread) [homeassistant.config_entries] Error importing platform config_flow from integration heatzy to set up heatzy configuration entry: No module named 'heatzypy'
```

**Comment by cyr-ius** [comment]
Fix  with ws-6.1.1.
Thank for a feedback

**Comment by luke7101** [comment]
Thanks !
